app/containers/Settings/Settings.jsx

```diff
@@ -116,141 +120,87 @@ export default class Settings extends Component<Props, State> {
-            <div className="itemTitle">Network</div>
-            <NetworkSwitch />
-          </div>
-
-          <div className="settingsItem">
-            <div className="itemTitle">Saved Wallet Accounts</div>
-            {map(accounts, account => (
-              <div className="walletList" key={`wallet${account.key}`}>
-                <div className="walletItem">
-                  <div className="walletName">{account.key.slice(0, 20)}</div>
-                  <div className="walletKey">{account.label}</div>
-                  <div
-                    className="deleteWallet"
-                    onClick={() =>
-                      this.deleteWalletAccount(account.label, account.key)
-                    }
-                  >
-                    <Delete />
-                  </div>
-                </div>
-              </div>
-            ))}
-          </div>
-          <Button onClick={() => this.saveWalletRecovery()}>
-            Export wallet recovery file
-          </Button>
-          <Button onClick={this.loadWalletRecovery}>
-            Load wallet recovery file
-          </Button>
-        </div>
-      </div>
+          <Tooltip title="Add Token" className={styles.headerButtonContainer}>
+            <AddIcon className={styles.add} />
+            <span>Add Token</span>
+          </Tooltip>
+        </UnderlinedHeader>
+        <Panel
+          className={styles.settingsPanel}
+          renderHeader={this.renderHeader}
+        >
+          <section className={styles.settingsItemsContainer}>
+            <SettingsItem title="THEME">
+              <SelectInput
+                items={Object.keys(themes)}
+                value={theme}
+                onChange={this.updateThemeSettings}
+              />
+            </SettingsItem>
+            <SettingsItem title="CURRENCY">
+              <SelectInput
+                items={Object.keys(CURRENCIES).map(value =>
+                  value.toUpperCase()
+                )}
+                value={currency.toUpperCase()}
+                onChange={this.updateCurrencySettings}
+                getItemValue={value => value.toLowerCase()}
+              />
+            </SettingsItem>
+            <SettingsItem title="BLOCK EXPLORER">
+              <SelectInput
+                items={Object.keys(EXPLORERS).map(value => EXPLORERS[value])}
+                value={explorer}
+                onChange={this.updateExplorerSettings}
+              />
+            </SettingsItem>
+            <SettingsLink to={ROUTES.ENCRYPT} title="ENCRYPT A KEY" />
+            <SettingsLink to={ROUTES.NODE_SELECT} title="NODE SELECTON" />
+            <WalletRecoveryPanel
+              title="WALLET RECOVERY"
+              loadWalletRecovery={this.loadWalletRecovery}
+              saveWalletRecovery={this.saveWalletRecovery}
+            />
+            <div className={styles.settingsDonations}>
+              Created by CoZ. Donations: Adr3XjZ5QDzVJrWvzmsTTchpLRRGSzgS5A
```

Comment:
Can we make this a link that loads up send container 🤓

Reply:
Sick idea, Working on it

Reply:
Hey man, managed to route it, but when clicking the link I lose the side bar navigation. Any ideas? see commit - https://github.com/CityOfZion/neon-wallet/pull/1365/commits/a0e16bafb5d228ddefcaf840bc682e2e34a7ff06